src/Templating/test/Templates.Test/RazorComponentsTemplateTest.cs

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.CommandLineUtils;
+using OpenQA.Selenium;
+using System.IO;
+using System.Net;
+using Templates.Test.Helpers;
+using Templates.Test.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Templates.Test
+{
+    public class RazorComponentsTemplateTest : BrowserTestBase
+    {
+        public RazorComponentsTemplateTest(BrowserFixture browserFixture, ITestOutputHelper output) : base(browserFixture, output)
+        {
+        }
+
+        [Fact]
+        public void RazorComponentsTemplateWorks()
+        {
+            RunDotNetNew("razorcomponents");
+
+            // We don't want the Directory.Build.props/targets interfering
+            File.WriteAllText(
+                Path.Combine(TemplateOutputDir, "Directory.Build.props"),
+                "<Project />");
+            File.WriteAllText(
+                Path.Combine(TemplateOutputDir, "Directory.Build.targets"),
+                "<Project />");
```

Comment:
It looks like, at the moment, the templates tests do some complicated stuff with `Directory.Build.props/targets` to work around the difference between the .NET Core SDK that we're building with versus the version that end users will install.

Unfortunately those workarounds aren't compatible with `netstandard2.0` projects, and the Razor Components templates contains a `netstandard2.0` project (alongside a `netcoreapp3.0` project). So this is my workaround for the workaround.

My hope is that all this complexity goes away when we move to newer versions of the .NET Core SDK and the Components packages are part of the shared framework natively.